### DISCUSSION TITLE
[BP#63] Ne pas assigner inutilement de valeur aux variables

### DISCUSSION THREAD

**Discussion by devcrafting** [cnumr/best-practices#70] [Bonnes pratiques] [answered]
Il me semble que la plupart des compilateurs vont savoir gérer cela. Reste à voir pour les languages interprétés mais j'ai un gros doute sur l'impact de cette BP.

**Comment by didierfred** [comment]
Je suis d'accord , avant qu'on voit un impact il faudrait déjà assigner beaucoup de variables !

**Comment by ACTLEM** [comment, marked as answer]
https://github.com/cnumr/best-practices/issues/127
